groups.yaml

```diff
@@ -70,3 +70,10 @@
     - GeoPandas
   shape: tri
   cmap: BrBG
+- name: Testing
+  items:
+    - Pytest
+    - Hypothesis
+    - Ghostwriter
```

Comment:
I'm guessing you are maintainer of ghostwriter ?

Reply:
Yeah, it's a CLI tool shipped with Hypothesis.